release/pypi/prep_binary_for_pypi.sh

```diff
@@ -52,14 +52,6 @@ for whl_file in "$@"; do
     (
         set -x
 
-        # Special build with pypi cudnn remove it from version
-        if [[ $whl_file == *"with.pypi.cudnn"* ]]; then
```

Comment:
I think both of these folders where removed from torch. Validated generated binary. It does not have any ``caffe2`` or ``/torch/lib/libnvrtc*``